src/content/polyfill.ts

```diff
@@ -1,5 +1,12 @@
 import type { MonetizationEventPayload } from '@/shared/messages';
 (function () {
+  const link = document.createElement('link');
+  if (link.relList.supports('monetization')) {
+    // eslint-disable-next-line no-console
+    console.debug('Monetization is already supported');
+    return;
+  }
```

Comment:
This shouldn't be needed as this is mostly an issue during development - live reload.

Reply:
This same issue will happen when extension updates from store, and sites might break (or we'll at least get unnecessary errors).

Reply:
Moved context invalidation related fixes out of scope of this PR:
https://github.com/interledger/web-monetization-extension/pull/538/commits/ce72dfbe826491efc7e9a7d1c0d369f8120db612